.github/workflows/svelte.yml

```diff
@@ -0,0 +1,37 @@
+name: Svelte
+
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./client/veryrezsi
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: npm ci
```

Comment:
I assume this whole file was coming from a template.
Anyway, for future reviewers:
  _npm ci_ is a nice touch (it is a faster npm install, only for this purpose)

Reply:
Yes it was a React template I used a while ago and I figured it would work for Svelte too. I did not find any good templates for SvelteKit yet.